_specs/ecip-_.md

```diff
@@ -0,0 +1,103 @@
+---
+ecip: TBD
+title: The Meta Fork ECIP: Establishing _Fork_ type ECIPs and their specifications and processes
+author: Mr. Meows D. Bits
+discussions-to: TBD
+status: WIP
```

Comment:
When this ECIP is accepted, we need to decide whether the status should be "Active" or "Final". I believe "Active" will be an appropriate status here as this is an ECIP about the ECIP process that may still be modified later.

Reply:
Agreed; `Active` seems sensible to me too.

Reply:
Resolved with https://github.com/ethereumclassic/ECIPs/pull/224/commits/b00bdcacc07bae5436a1444f2c781c8f49508fce